index.html

```diff
@@ -191,6 +275,70 @@
       </div>
     </div>
 
-  
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js"></script>
+    <script src="bower_components/jquery.hotkeys/jquery.hotkeys.js"></script>
```

Comment:
@steveathon I'm in the process of submitting a PR to the [cdnjs.com repo](https://github.com/cdnjs/cdnjs) containing the jQuery.hotkeys library. As soon as I get it done and it's approved I'll update my code to use the CDN link and I'll remove Bower while I'm at it.